blog/serializers.py

```diff
@@ -23,7 +23,7 @@ class BlogSerializer(serializers.ModelSerializer):
     # comment = CommentSerializer(many=True,read_only=True)
     class Meta:
         model = Stori
-        fields = ['id', 'title', 'slug', 'description', 'content', 'status', 'category']
+        fields = ['id', 'title', 'slug', 'description', 'content', 'status', 'featured', 'category']
```

Comment:
The corresponding update to the BlogSerializer ensures that the new featured field is properly serialized and available in API responses.